planty/users/views.py

```diff
@@ -1,21 +1,19 @@
 from django.shortcuts import render, redirect
 from .forms import UserRegisterForm
+from django.contrib import messages
 
 
 def register(request):
     if request.method == "POST":
         form = UserRegisterForm(request.POST)
         if form.is_valid():
+            username = form.cleaned_data.get('username')
             form.save()
-            # TODO add a success message
+            messages.success(request, f"Account for {username} is created")
             return redirect('login')
         else:
-            # TODO add an error message
-            pass
+            messages.error(request, "Invalid form")
+            form = UserRegisterForm()
```

Comment:
Do we need to re-initialize the form here? Doesn't that mean that the user will see an empty form again? If we remove this line, is an invalid form still populated after an error?